### DISCUSSION TITLE
Content Security Policy

### DISCUSSION THREAD

**Discussion by douglaseggleton** [stitchesjs/stitches#862] [Help]
Is stitches able to support the use of a content-security policy?

**Comment by jnlsn** [comment]
I'd also love to know more about this. You can pretty easily use a nonce for SSR styles but not sure where to go from here on getting this to work with dynamic css.
```javascript
<style
  nonce="#####"
  dangerouslySetInnerHTML={{ __html: getCssText() }}
/>
```
